tools/profiler/finagle-with-svg.py

```diff
@@ -0,0 +1,31 @@
+import os
+import sys
+
+# Takes in a flame graph svg that is scaled by 1000 and prints a version with fixed cycles.
+def main(svg_in):
```

Comment:
maybe you can come up with a better name for this script that contains a bit of the description?

Reply:
Changed to `adjust-scaled-flame-svg.py`!